public/helpers/update-users.js

```diff
@@ -0,0 +1,34 @@
+const updateUsers = (sender, users) => {
```

Comment:
Can we change `users` to `participants` everywhere in this file, so that it is more in line with how we are referring to people in Noga's designs?